cmd/zed/zed.d/zed.rc

```diff
@@ -86,6 +86,14 @@
 #ZED_SPARE_ON_IO_ERRORS=1
 
 ##
+# Turn on/off enclosure LEDs when drives get DEGRADED/FAULTED.  This works for
+# device mapper and multipath devices as well.  Your enclosure must be
+# be supported by the Linux SES driver for this to work.
+#
+#ZED_USE_ENCLOSURE_LEDS=1
```

Comment:
What is the consequence of trying to change the lights if it is unsupported? Put differently, is it safe to enable this by default? I realize ZoL might default to off for everything, but if it's safe, I will likely propose this be enabled-by-default in Ubuntu.